curvefs/conf/client.conf

```diff
@@ -52,7 +52,10 @@ excutorOpt.maxRPCTimeoutMS=8000
 excutorOpt.maxRetrySleepIntervalUS=8000000
 excutorOpt.minRetryTimesForceTimeoutBackoff=5
 excutorOpt.maxRetryTimesBeforeConsiderSuspend=20
-excutorOpt.batchLimit=10000
+# batch limit of get inode attr and xattr
+excutorOpt.batchInodeAttrLimit=10000
```

Comment:
excutorOpt is common， not only for inode， this config option seems not belong to the excutor freamwork